client/landing/stepper/declarative-flow/flows/site-setup-flow/site-setup-flow.ts

```diff
@@ -91,6 +92,48 @@ const siteSetupFlow: Flow = {
 
 		return steps;
 	},
+	getCustomInitialStep( { queryParams, steps } ) {
```

Comment:
@daledupreez Do you have any specific thoughts about having this function receive the queryParams instead of using it directly from the window.location.search ?

Reply:
My main thought was that it would be more straightforward to test if we supply the URL parameters as an argument and would be a minor ergonomic benefit to most implementations of the code, as they won't need to extract the URL parameters. I am not super-strongly in favour of keeping this, though.